snmp/datadog_checks/snmp/data/default_profiles/_hp-compaq-health.yaml

```
@@ -159,6 +159,7 @@ metrics:
     symbol:
       OID: 1.3.6.1.4.1.232.6.2.6.1.0
       name: cpqHeThermalCondition
+    # NOTE: other(1), notSupported(2), notInstalled(3), installed(4)
   - MIB: CPQHLTH-MIB
```

```suggestion
  - # NOTE: other(1), notSupported(2), notInstalled(3), installed(4)
    MIB: CPQHLTH-MIB
```

To make it more explicit that the notes belongs to